logstash-core/src/test/resources/org/logstash/config/ir/complex.cfg

```diff
@@ -0,0 +1,1999 @@
+# Generated by Chef for REDACTED
```

Comment:
To merge this we'll need https://github.com/elastic/logstash/issues/7403#issuecomment-346319046 addressed since this isn't under our CLA yet.

Reply:
@andrewvc I can just delete this now and bring it back when and if we get that CLA. If we don't I can spend some time to reproduce this with a minimal example (it's like a 1h effort probably).